hi-ml-azure/src/health_azure/himl.py

```diff
@@ -221,17 +221,21 @@ def create_run_configuration(workspace: Workspace,
     return run_config
 
 
-def create_crossval_hyperdrive_config(num_splits: int,
-                                      cross_val_index_arg_name: str = "crossval_index",
-                                      metric_name: str = "val/loss") -> HyperDriveConfig:
+def create_grid_hyperdrive_config(values: List[str],
+                                  argument_name: str,
+                                  metric_name: str) -> HyperDriveConfig:
     """
-    Creates an Azure ML HyperDriveConfig object for running cross validation. Note: this config expects a metric
-    named <metric_name> to be logged in your training script([see here](
+    Creates an Azure ML HyperDriveConfig object that runs a simple grid search. The Hyperdrive job will run one child
+    job for each of the values provided in `values`, and each child job will have a suffix added to the commandline
+    like `--argument_name value`.
+
+    Note: this config expects that a metric is logged in your training script([see here](
     https://docs.microsoft.com/en-us/azure/machine-learning/how-to-tune-hyperparameters#log-metrics-for-hyperparameter-tuning))
+    that will be monitored by Hyperdrive. The name of this metric is given by `metric_name`.
 
-    :param num_splits: The number of splits for k-fold cross validation
-    :param cross_val_index_arg_name: The name of the commandline argument that each of the child runs gets, to
-        indicate which split they should work on.
+    :param values: The list of values for which  number of splits for k-fold cross validation
```

Comment:
Not sure I understand this definition - I think the num splits bit is leftover from the previous definition?

Reply:
Is this in fact the list of values that the argument 'argument_name' can take one of?

Reply:
sorry, yes, leftover. cleaned up now.